### PR TITLE
opendiff: add page, history: add page; scripts/pdf/*: add css page

### PR DESCRIPTION
**Pull request by v-sastry**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/osx/opendiff.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+> Built-in command of OS X.
+> More information: <https://www.manpagez.com/man/1/opendiff/>.
+
+- Compare two files:
+
+`opendiff {{path/to/file1}} {{path/to/file2}}`
+
+- Compare two directories:
+
+`opendiff {{path/to/dir1/}} {{path/to/dir2/}}`
```
Comment (kbdharun) [inline pages/osx/opendiff.md:13]:
```suggestion
`opendiff {{path/to/directrory1}} {{path/to/directory2}}`
```
Would this be better?

File: scripts/pdf/README.md [hunk 2]
```diff
@@ -34,5 +34,7 @@ Complete information about the arguments can be viewed by running:
 The color-schemes that can be specified are:
```
Comment (kbdharun) [inline scripts/pdf/README.md:35]: If you don't mind can you add images of the themes in README too? (Adding this for future reference, I think we can add the images to `images` directory in the repo instead of using the current one)

**Review by kbdharun — APPROVED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page. 

> It's nice to see new themes for PDFs. Can you add some images for our reference to check out the themes?

File: scripts/pdf/high-contrast-dark.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: MIT */
+
+code {
+	color: #FFD042;
+  }
+
+  h1, h2, h4, ul {
```
Comment (navarroaxel) [inline scripts/pdf/high-contrast-dark.css:7]: please fix the indentation

File: pages/osx/opendiff.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# opendiff
```
Comment (navarroaxel) [inline pages/osx/opendiff.md:1]: please move this new pages to a new PR

File: pages/osx/history.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`history`
+
+- Clear the history list:
+
+`history -c`
+
+- Append history lines entered since the beginning of the current Bash session:
+
+`history -a`
```
Comment (EmilySeville7cfg) [inline pages/osx/history.md:16]:
```suggestion
- [a]ppend history lines entered since the beginning of the current Bash session:

`history -a`
```

The same issue as above.

File: pages/osx/history.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Append the history lines not already read from the history file to the current history list:
+
+`history -n`
```
Comment (EmilySeville7cfg) [inline pages/osx/history.md:20]:
```suggestion
- Append the history lines [n]ot already read from the history file to the current history list:

`history -n`
```

The same issue as above.

File: pages/osx/history.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- Read the current history file and append its contents to the history list:
+
+`history -r`
```
Comment (EmilySeville7cfg) [inline pages/osx/history.md:24]:
```suggestion
- Read the current history file and append its contents to the history list:

`history -r`
```

The same issue as above.

File: pages/osx/opendiff.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages/osx/opendiff.md:13]: Warning: this change may conflict with #9578 PR. Please check it before suggesting PR which fixes placeholders.

File: scripts/pdf/high-contrast-dark.css [hunk 8]
```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: MIT */
```
Comment (EmilySeville7cfg) [inline scripts/pdf/high-contrast-dark.css:1]: Optional fix: separate PR into 2 ones based on directories changed: everything in `scripts/pdf` can be put in a separate PR.

File: scripts/pdf/README.md [hunk 9]
```diff
@@ -34,5 +34,7 @@ Complete information about the arguments can be viewed by running:
...
```
Comment (EmilySeville7cfg) [inline scripts/pdf/README.md:34]: Optional fix: separate PR into 2 ones based on directories changed: everything in `scripts/pdf` can be put in a separate PR.

File: scripts/pdf/high-contrast-light.css [hunk 10]
```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: MIT */
```
Comment (EmilySeville7cfg) [inline scripts/pdf/high-contrast-light.css:1]: Optional fix: separate PR into 2 ones based on directories changed: everything in `scripts/pdf` can be put in a separate PR.

**Review by EmilySeville7cfg — APPROVED** [review]
- No mandatory changes found, approved.
- Thanks for a contribution! ❤️

**Comment by kbdharun** [comment]
Can this PR be merged?

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by Waples** [comment]
Stale: to many things that are not fixed. Closing this.
